web/index.php

```diff
@@ -101,13 +101,19 @@
 $app->container->set('access_token', isset($_SESSION['access_token']) ? $_SESSION['access_token'] : null);
 
 $app->container->singleton(\Application\CacheService::class, function ($container) {
-    $redisServer = null;
+
+    $redis = $container->settings['custom']['redis'];
+
     if ($host = getenv('REDIS_HOST')) {
-        $redisServer = "tcp://$host:6379";
+        $redis['scheme'] = 'tcp';
+        $redis['host'] = $host;
```

Comment:
Why not set the host only here? After all the others are already preset, aren't they?

Reply:
On platform we're just copying over the config file and it pulls in special variables.

Reply:
Find. But if we only set the host here there should only be the host variable set in this if statement. The port and the scheme should not be set here.

Reply:
I agree, setting scheme and port here is redunant

Reply:
Never mind my remark, for breaking changes sake setting scheme and port should be done.

Reply:
redis config can be set several ways from the config either string or array and can be TCP or Unix sockets.  If it's changed in the dist I didn't want that to effect REDIS_HOST if that is the override. I put the host back to being a string. For example if we set in the dist to use Unix sockets instead of TCP,  scheme, and port wouldn't be available properly if someone was using docker and overrides it.